<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeAdminPassWindow</class>
 <widget class="QDialog" name="ChangeAdminPassWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>383</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>200</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="styleSheet">
         <string notr="true">background: white</string>
        </property>
        <property name="text">
         <string>Измените пароль</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="passwordEdit"/>
      </item>
      <item>
       <widget class="QCheckBox" name="showPasswordCheckbox">
        <property name="styleSheet">
         <string notr="true">background: white</string>
        </property>
        <property name="text">
         <string>Показать пароль</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="errorLabel">
        <property name="styleSheet">
         <string notr="true">background: white</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Изменить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
